content-src/components/DiscoveryStreamComponents/DSFooter/DSFooter.jsx

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { cardContextTypes } from "../../Card/types.js";
+import React from "react";
+
+const StatusMessage = ({ icon, fluentID }) => (
+  <div className="status-message">
+    <span
+      aria-haspopup="true"
+      className={`story-badge-icon icon icon-${icon}`}
+    />
+    <div className="story-context-label" data-l10n-id={fluentID} />
+  </div>
+);
+
+export class DSFooter extends React.PureComponent {
+  static getDerivedStateFromProps(nextProps, prevState) {
+    return {
+      hasUpdate: prevState && prevState.badge !== nextProps.badge,
+      ...nextProps,
+    };
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    if (this.state.badge !== prevState.badge) {
+      // eslint-disable-next-line react/no-did-update-set-state
+      this.setState({ hasUpdate: false });
+    }
+  }
+
+  render() {
+    const { context, badge, hasUpdate } = this.state;
+    const { icon, fluentID } = cardContextTypes[badge] || {};
+
+    return (
+      <footer className="story-footer">
```

Comment:
Should change it to use div. Footer element might not make sense for [accessibility which recommends ](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Contentinfo_role)having one per webpage

Reply:
Good catch!